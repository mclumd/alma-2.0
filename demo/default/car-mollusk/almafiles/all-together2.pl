rel(mollusk, steve).
if(now(2), rel(cephalopod, steve)).
if(now(7), rel(nautilus, steve)).
if(now(12), rel(naked_nautilus, steve)).

obs(quote(rel(gas_tank_car, ford))).
obs(quote(not(rel(fueled, ford)))).

fif(now(5), obs(quote(rel(electric_car, prius)))).
fif(now(5), obs(quote(rel(gas_tank_car, prius)))).
fif(now(5), obs(quote(rel(charged, prius)))).
fif(now(5), obs(quote(not(rel(fueled, prius))))).
