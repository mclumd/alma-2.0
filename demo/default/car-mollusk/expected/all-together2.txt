~rel(shell_bearer, steve)
rel(naked_nautilus, steve)
rel(nautilus, steve)
rel(cephalopod, steve)
rel(mollusk, steve)
abnormal(steve, mollusk, "~rel(shell_bearer, steve)")
abnormal(steve, nautilus, "~rel(shell_bearer, steve)")
rel(car, ford)
rel(gas_tank_car, ford)
~rel(powered, ford)
~rel(fueled, ford)
abnormal(ford, car, "~rel(powered, ford)")
rel(car, prius)
rel(gas_tank_car, prius)
rel(electric_car, prius)
rel(charged, prius)
~rel(fueled, prius)
rel(powered, prius)
abnormal(prius, gas_tank_car, "rel(powered, prius)")
